website/static/js/fangorn.js

```diff
@@ -833,6 +836,45 @@ function expandStateLoad(item) {
     reapplyTooltips();
 }
 
+/**
+ * @param tree A Treebeard _item object for the row
+ * @param nodeID Current node._id
+ * @param file window.contextVars.file object
+ */
+function findCurrentFileID(tree, nodeID, file) {
+    var tb = this;
+    if (tb.options.folderIndex !== undefined && tb.options.folderArray !== undefined && tb.options.folderIndex < tb.options.folderArray.length) {
```

Comment:
Would be nice if there was an initialization hook:

```js
var tb = new Treebeard({
    init: function() {
        this._fangorn = {};
    }
});
```

@caneruguz Is something like this available?